classes/sdcard_image-sunxi.bbclass

```diff
@@ -35,10 +35,9 @@ IMAGE_DEPENDS_sunxi-sdimg += " \
 			dosfstools-native \
 			virtual/kernel \
 			virtual/bootloader \
-                        sunxi-board-fex \
 			"
 
-rootfs[depends] += "virtual/kernel:do_deploy sunxi-board-fex:do_deploy"
+rootfs[depends] += "virtual/kernel:do_deploy"
```

Comment:
I also looked at this during the weekend and I have the suspiction that the above line doesn't actually do anything. I think the correct syntax would be do_rootfs[depends] += .... I'd say it will be okay to remove it completely, as in image.bbclass we have do_build[depends] += "virtual/kernel:do_deploy" which should be enough.